File: coreui-free-angular-admin-template-main/src/app/models/CheckedMatchDTO.ts
//pour afficher les cv cochés dans la liste des cv cochés
export interface CheckedMatchDTO {
  id: number;
  offreId: number | null;
  offreTitre: string | null;

  candidatureId: number | null;
  candidatNom: string | null;
  candidatPrenom: string | null;
  candidatEmail: string | null;

  recruteurEmail: string | null;
  scoreOverall: number | null;
  filenameSnapshot: string | null;
  createdAt: string; // ISO string
}

File: stage/src/main/java/com/matcheval/stage/dto/CheckedMatchDTO.java
// src/main/java/com/matcheval/stage/dto/CheckedMatchDTO.java
package com.matcheval.stage.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import java.time.LocalDateTime;

@Data
@AllArgsConstructor
public class CheckedMatchDTO {
    private Long id;
    private Long offreId;
    private String offreTitre;

    private Long candidatureId;
    private String candidatNom;
    private String candidatPrenom;
    private String candidatEmail;

    private String recruteurEmail;
    private Double scoreOverall;
    private String filenameSnapshot;
    private LocalDateTime createdAt;
}
